app/editor/src/features/content/form/hooks/useContentForm.ts

```diff
@@ -224,6 +226,14 @@ export const useContentForm = ({
     }
   }, [getStream, fileReference?.contentType, path]);
 
+  React.useEffect(() => {
```

Comment:
What's the difference between AvStream and Stream?

Reply:
confirmed, no need to have this, callback could handle changes.